### ISSUE TITLE
How can add style??

### ISSUE THREAD

**Issue by brayan15** [nigma/django-easy-pdf#49]
I added some styles, but I cant' see on pdf

**Comment by aureliendebord** [comment]
If you are using a template to render your PDF, you have to put your styles in your template after the </head> tag. See the doc [here](https://django-easy-pdf.readthedocs.io/en/v0.2.0-dev1/usage.html#prepare-html-templates).

**Comment by brayan15** [comment]
Hello, I did that, but I can't see some styles, for example if I added background, I can't see that background on pdf file

**Comment by aureliendebord** [comment]
Hey Brayan,
Can you paste your template code here?

**Comment by brayan15** [comment]
sure, 
{% extends "easy_pdf/base.html" %}
{% block content %}
  <div id="content">
    some content here with background
  </div>
{% endblock %}

{% block extra_style %}
<style>
  #content{
    background: red;
    width: 200px;
    height: 200px;
  }
</style>
{% endblock %}

I tried with this beta code, but I never see changes on pdf

**Comment by aureliendebord** [comment]
It will be better with 
`background-color: red;`

**Comment by brayan15** [comment]
it's work, but some styles doesn't work for example, margin and  padding

**Comment by aureliendebord** [comment]
I just have tested on my code with a `margin-top` and it works.
Can you paste some code?

**Comment by brayan15** [comment]
How can call images on my pdf file?

That happened when I try to call images from static

Invalid block tag on line 106: 'static', expected 'endblock'. Did you forget to register or load this tag?

**Comment by aureliendebord** [comment]
Can you paste your code?

You can follow my thread about displaying images #48

**Comment by brayan15** [comment]
<img src="{% static "images/img.jpg" %}" alt="">

**Comment by aureliendebord** [comment]
Did you try:

**template.html**
`<img src="images/img.jpg" />`

...assuming you declared a STATIC_ROOT

**Comment by brayan15** [comment]
Yeah, I declared, but with static doesn't work, but if I put that on base template, work, but if I put it inside block does't work, so I decided put on base template and worked  it.

**Comment by aureliendebord** [comment]
You're lucky because I can't display images at all...

**Comment by brayan15** [comment]
amm I also ran `python manage.py collectstatic`, maybe lack run this command.

**Comment by brayan15** [comment]
Is it possible put image like background? I tried but I got an error when I did it. Ty

**Comment by davidfraga** [comment]
Brayan, how can you put on the base template? I'm trying to put a background-image but I think the url address is wrong
